Framework/Core/include/Framework/ParamRetriever.h

```diff
@@ -0,0 +1,34 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef FRAMEWORK_PARAMETERRETRIEVER_H
+#define FRAMEWORK_PARAMETERRETRIEVER_H
+
+#include <string>
```

Comment:
need also `#include <vector>`

Reply:
yeah, not sure why it works on my mac.. Fixed.